.github/workflows/errno.yml

```diff
@@ -0,0 +1,32 @@
+name: Error numbers
+on:
+  pull_request:
+    paths:
+      - 'lib/libc/minimal/include/errno.h'
+
+jobs:
+  check-errno:
+    runs-on: ubuntu-latest
+    container:
+      image: zephyrprojectrtos/ci:v0.17.1
+      options: '--entrypoint /bin/bash'
+    steps:
+      - name: Update PATH for west
+        run: |
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: west setup
+        run: |
+          west init -l . || true
+          west update
```

Comment:
since we are just checking repo files, west init/update is not needed

Reply:
While you are completely correct, I'd rather leave this in for consistency with other actions. This action only ever runs if `errno.h` is modified, so the overhead is very small.

Reply:
this means we're downloading ~2GB of dependencies for nothing every time

Reply:
we can save > 2min, so I think it definitely needs to be removed

![image](https://user-images.githubusercontent.com/25011557/118275558-dda47900-b4c6-11eb-9bab-e00fc9d3cea2.png)

Reply:
agree, we do not need to pull the work just to be checking local files